Makefile

```diff
@@ -145,7 +145,7 @@ edit-cypress-secrets: read-keyvault-config install-fetch-config azure-login ## E
 .PHONY: shell
 shell: ## Open a shell on the app instance on PaaS, eg: make qa shell
 	cf target -s ${SPACE}
-	cf ssh apply-clock-${APP_NAME_SUFFIX} -t -c 'cd /app && /usr/local/bin/bundle exec rails c'
+	cf ssh apply-${APP_NAME_SUFFIX} -t -c 'cd /app && /usr/local/bin/bundle exec rails c'
```

Comment:
We initially went with clock as this avoids us hooking into the app containers, which if we cause memory issues there could bring down an environment like qa (as it doesn't have many instances up) or be problematic for users in production. I'd suggest either bumping the memory in test envs or hooking into workers instead

Reply:
The `apply-clock-qa` currently has 0.5 GB allocated memory, aligning this with `apply-qa` would only require an increase of 0.5GB so that is perhaps a simpler and safer solution.  The cost is ~£13 per month for 3 environments (assuming we don't need this for review apps) so not unreasonable if it stops us inadvertently overloading production.

Reply:
Thank you all for your comments. 🙏  Should I close this PR then?

Reply:
Yes please I think we can bump up the memory in a different one once agreed, thanks!